### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by danielpassos**
Relates to:

* [AGDROID-590](https://issues.jboss.org/browse/AGDROID-590) - Add Badge in Github README
* [AGDROID-593](https://issues.jboss.org/browse/AGDROID-593) - Update library version in Github README 
* [AGDROID-594](https://issues.jboss.org/browse/AGDROID-594) - Update documentation link in Github README 
* [AGDROID-595](https://issues.jboss.org/browse/AGDROID-595) - Update demo apps link in Github README

### CODE REVIEW

**Comment by danielpassos** [comment]
@secondsun @corinnekrych 👀 ?

File: README.md [hunk 1]
```diff
@@ -26,37 +32,37 @@ There are two supported ways of developing apps using AeroGear for Android: Andr
...
-  <version>3.0.0</version>
+  <version>3.1.0</version>
   <type>aar</type>
 </dependency>
 ```
 
 ## Documentation
 
-For more details about the current release, please consult [our documentation](http://aerogear.org/docs/guides/aerogear-android/).
+For more details about the current release, please consult [our documentation](https://aerogear.org/android/).
```
Comment (jcesarmobile) [inline README.md]: You changed the link here for a new one, but there is another `http://aerogear.org/docs/guides/aerogear-android/` in the table at the beginning. Should that be changed with this new URL too?

Comment (danielpassos) [inline README.md]: Done!

**Comment by jcesarmobile** [comment]
LGTM

**Pull request merged**